app/controllers/orgs/rosters_controller.rb

```diff
@@ -228,6 +311,88 @@ def get_user_to_group_hash(grouping)
 
       mapping
     end
+
+    # Returns name of the linked google course to current organization (for syncing rosters)
+    def current_organization_google_course_name
+      return nil if current_organization.google_course_id.nil?
+      authorize_google_classroom
+      course = @google_classroom_service.get_course(current_organization.google_course_id)
+      return course.name unless course.nil?
+      nil
+    rescue Google::Apis::Error
+      nil
+    end
+
+    # Returns list of students in a google classroom with error checking
+    # rubocop:disable Metrics/MethodLength
+    def list_google_classroom_students(course_id)
+      response = @google_classroom_service.list_course_students(course_id)
+      response.students ||= [] # Set to empty array if no students in course
```

Comment:
Do you mind adding a slightly more explicit `response.students` line under this to show that's the return value? it's a little tricky following how `response.students ||= []` returns response.students if it exists.